webpack et al need to be in main dependencies

Because `html-webpack-plugin` and so forth are in this package's `devDependencies`, they're not installed when I add this package as a dep of my project.
```
$ yarn add --dev module-docs
andy@Andys-MBP-2:clarity (master) $ module-docs start
module.js:549
    throw err;
    ^

Error: Cannot find module 'html-webpack-plugin'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/andy/clarity/node_modules/module-docs/lib/webpack.config.js:5:27)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

--- a/nodemon.json
+++ b/nodemon.json
@@ -1,0 +1,3 @@
+{
+  "ext": "js, jsx, css, html"
+}

--- a/package.json
+++ b/package.json
@@ -26,10 +26,13 @@
     "@emotion/core": "^10.0.6",
     "@emotion/styled": "^10.0.6",
     "@material-ui/styles": "^3.0.0-alpha.7",
+    "babel-loader": "^8.0.5",
     "body-parser": "^1.18.3",
+    "css-loader": "^2.1.0",
     "commander": "^2.19.0",
     "concurrently": "^4.1.0",
     "express": "^4.16.4",
+    "html-webpack-plugin": "^3.2.0",
     "idb-keyval": "^3.1.0",
     "normalize.css": "^8.0.1",
     "prismjs": "^1.15.0",
@@ -38,7 +41,12 @@
     "react-autosuggest": "^9.4.3",
     "react-dom": "^16.8.0-alpha.0",
     "react-markdown": "^4.0.6",
-    "react-router-dom": "^4.3.1"
+    "react-router-dom": "^4.3.1",
+    "style-loader": "^0.23.1",
+    "webpack": "^4.28.4",
+    "webpack-cli": "^3.2.1",
+    "webpack-dev-middleware": "^3.5.0",
+    "webpack-dev-server": "^3.1.14"
   },
   "devDependencies": {
     "@babel/cli": "^7.2.3",
@@ -50,8 +58,6 @@
     "@babel/preset-env": "^7.2.3",
     "@babel/preset-react": "^7.0.0",
     "babel-eslint": "^10.0.1",
-    "babel-loader": "^8.0.5",
-    "css-loader": "^2.1.0",
     "eslint": "^5.12.0",
     "eslint-config-node": "^4.0.0",
     "eslint-config-prettier": "^3.3.0",
@@ -61,13 +67,7 @@
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-react": "^7.12.3",
     "eslint-plugin-standard": "^4.0.0",
-    "html-webpack-plugin": "^3.2.0",
     "nodemon": "^1.18.9",
-    "prettier": "^1.15.3",
-    "style-loader": "^0.23.1",
-    "webpack": "^4.28.4",
-    "webpack-cli": "^3.2.1",
-    "webpack-dev-middleware": "^3.5.0",
-    "webpack-dev-server": "^3.1.14"
+    "prettier": "^1.15.3"
   }
 }

--- a/src/client/Main.jsx
+++ b/src/client/Main.jsx
@@ -85,11 +85,10 @@ const Main = ({ pkg }) => {
   )
 
   const handleClick = child => {
+    console.log(child.content)
     setSource(child.content)
     setInfo(JSON.parse(child.info))
   }
-
-  console.log(pkg)
 
   return (
     <div className={classes.root}>

--- a/src/server/serve.js
+++ b/src/server/serve.js
@@ -20,7 +20,6 @@ module.exports = (modulePath, config) => {
   )
 
   app.post("/modules", FileController.getFiles(modulePath, config))
-
   app.post("/module/:name", FileController.getPackage(modulePath))
 
   app.get("*", function response(req, res) {

